src/TxHistory/TxDetails/TxDetails.tsx

```diff
@@ -69,6 +69,12 @@ export const TxDetails = () => {
           </Boundary>
         </Banner>
 
+        <View style={styles.borderTop}>
+          <Label>{strings.memo}</Label>
+        </View>
+
+        <Text style={styles.assetsTitle}>{transaction.memo}</Text>
+
```

Comment:
I asked product about what happens when the memo is empty. Should the field be hidden? to show empty field? default text ? etc